CONTRIBUTING.md

```diff
@@ -142,6 +142,15 @@ cd docs/_build/html/
 open index.html
 ```
 
+## Write integration tests
+1. Create a class that inherits `IntegrationTestCase` from `tests.integration.integration_test_case` to store all individual tests under (ex: `class TestWallet(IntegrationTestCase)`)
```

Comment:
This should note that this is only necessary if adding functionality to a new part of the library